api/envoy/api/v2/route/route.proto

```diff
@@ -609,6 +609,12 @@ message RouteAction {
   CorsPolicy cors = 17;
 
   reserved 21;
+
+  // If present, and the request is a gRPC request, use the 'grpc-timeout' header, or its
```

Comment:
I think this comment got lost, but can we link to somewhere that specifies the format of the grpc-timeout header? Whatever you think is best is fine.

Reply:
Done.